kratos/solving_strategies/schemes/residual_based_bossak_displacement_scheme.hpp

```diff
@@ -376,10 +379,24 @@ class ResidualBasedBossakDisplacementScheme
 
         // Check for minimum value of the buffer index
         // Verify buffer size
-        KRATOS_ERROR_IF(rModelPart.GetBufferSize() < 2) << "Insufficient buffer size. Buffer size should be greater than 2. Current size is" << rModelPart.GetBufferSize() << std::endl;
+        KRATOS_ERROR_IF(rModelPart.GetBufferSize() < 2)
+            << "Insufficient buffer size. Buffer size should be greater than 2. Current size is: "
+            << rModelPart.GetBufferSize() << std::endl;
 
         // Check for admissible value of the AlphaBossak
-        KRATOS_ERROR_IF(mAlpha.m > 0.0 || mAlpha.m < -0.3) << "Value not admissible for AlphaBossak. Admissible values should be between 0.0 and -0.3. Current value is " << mAlpha.m << std::endl;
+        KRATOS_ERROR_IF(mBossak.alpha > 0.0 || mBossak.alpha < -0.5) << "Value not admissible for "
+            << "AlphaBossak. Admissible values are between 0.0 and -0.5\nCurrent value is: "
+            << mBossak.alpha << std::endl;
+
+        const double epsilon = std::numeric_limits<double>::epsilon();
```

Comment:
This isn't very robust. There have been discussions about this issue but I don't know if we have any existing utility to do this. See https://github.com/chromium/chromium/blob/master/cc/base/math_util.cc for an example of how it can be done.

Reply:
which function are you referring to?

Reply:
IsNearlyTheSame

Reply:
done.